code/Language/Drasil/SentenceExtract.hs

```diff
@@ -0,0 +1,129 @@
+module Language.Drasil.SentenceExtract (getDoc)where
```

Comment:
Should be merged into `Language.Drasil.Document.Extract`.

Reply:
This file has been merged into `Language.Drasil.Document.Extract` in 8c1738588678f66ed29a6da31c4b7a6f243eaabe.